board-viewer.html

```diff
@@ -895,6 +893,23 @@
       }
     }
 
+    /* If the match is an array, we can assume that is a LIST type filter */
```

Comment:
I think this way of adding LIST type filter can be more useful and simplify the current calls, we only need to pass directly the array instead of pass ```{type: 'LIST', set: []}```